main.py

```diff
@@ -15,26 +30,138 @@ def print_board(board):
+                tempX = x
+                for i in range(ship.hitPoints):
+                    # check if there's an existing ship here
+                    if board[tempX][y] == empty_space:
+                        room = True
+                    else:
+                        room = False
+                        break
+                    tempX+=1
+                if room == True:
+                    tempX = x
+                    # place ship
+                    for i in range(ship.hitPoints):
+                        board[tempX][y] = ship.letter
+                        tempX+=1
+
+            # is there room for the ship to the left of the coordinate?
+            elif y - ship.hitPoints - 1 >= 0:
+                tempY = y
+                for i in range(ship.hitPoints):
+                    if board[x][y] == empty_space:
+                        room = True
+                    else:
+                        room = False
+                        break
+                    tempY-=1
+                if room == True:
+                    tempY = y
+                    for i in range(ship.hitPoints):
+                        board[x][tempY] = ship.letter
+                        tempY-=1
+
+            # is there room for the ship above the coordinate?
+            elif x - ship.hitPoints - 1 >= 0:
+                tempX = x
+                for i in range(ship.hitPoints):
+                    if board[tempX][y] == empty_space:
+                        room = True
+                    else:
+                        room = False
+                        break
+                    tempX-=1
+                if room == True:
+                    tempX = x
+                    for i in range(ship.hitPoints):
+                        board[tempX][y] = ship.letter
+                        tempX-=1
+
+            # is there room for the ship to the right of the coordinate?
+            elif y + ship.hitPoints - 1 < board_size:
+                tempY = y
+                for i in range(ship.hitPoints):
+                    if board[x][tempY] == empty_space:
+                        room = True
+                    else:
+                        room = False
+                        break
+                    y+=1
+                if room == True:
+                    tempY = y
+                    for i in range(ship.hitPoints):
+                        board[x][tempY] = ship.letter
+                        tempY+=1
+
+def createShips():
+    playerShips.append(Ship("Carrier", 5, "C"))
+    playerShips.append(Ship("Battleship", 4, "B"))
+    playerShips.append(Ship("Cruiser", 3, "C"))
+    playerShips.append(Ship("Submarine", 3, "S"))
+    playerShips.append(Ship("Destroyer", 2, "D"))
+    botShips.append(Ship("Carrier", 5, "C"))
+    botShips.append(Ship("Battleship", 4, "B"))
+    botShips.append(Ship("Cruiser", 3, "C"))
+    botShips.append(Ship("Submarine", 3, "S"))
+    botShips.append(Ship("Destroyer", 2, "D"))
```

Comment:
much prefer this use case. cleaner, shorter and eliminates the need for magic numbers

`SHIP_TYPES = [
("Carrier", 5, "C"),
("Battleship", 4, "B"),
("Cruiser", 3, "C"),
("Submarine", 3, "S"),
("Destroyer", 2, "D"),
]

def createShips():
for ship_class, hit_points, letter in SHIP_TYPES:
playerShips.append(Ship(ship_class, hit_points, letter))
botShips.append(Ship(ship_class, hit_points, letter))
`